websites/A/AtCoder/presence.ts

```diff
@@ -1,15 +1,15 @@
 const presence = new Presence({
     clientId: "845360129715994685"
   }),
-  timeElapsed: number = ~~Date.now() / 1000;
+  timeElapsed: number = Date.now() / 1000;
```

Comment:
when i use `~~Date.now() / 1000` the presence is not showing 

![image](https://user-images.githubusercontent.com/57343545/119840745-d5712280-bf37-11eb-8db2-544656e03e85.png)

Reply:
It should be `~~(Date.now() / 1000)`

Reply:
oooo thank u iu!!